src/Eagle.ts

```diff
@@ -4268,12 +4263,12 @@ export class Eagle {
 
         setTimeout(function() {
             //handling selecting and highlighting the newly created node
-            const clickTarget = $($("#paramsTableWrapper tbody").children()[fieldIndex]).find('.selectionTargets')[0]
+            const clickTarget = $($(".paramsTableWrapper tbody").children()[fieldIndex]).find('.selectionTargets')[0]
```

Comment:
This comment actually refers to Eagle.ts line 4259.

There are a few errors in the comment: "cell" is misspelled, and the end of the comment seems to be missing.